crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,305 @@
+function CrowdsourceHinter(runtime, element, data){
+
+    var executeHinter = true;
```

Comment:
Comment what executeHinter does, or better yet, rename it to something semantic. E.g.

    var onHinterPage; // We don't do hinter logic if we're on a different tab in a sequential.